xrpackage.js

```diff
@@ -1850,6 +1864,12 @@ export class XRPackage extends XRNode {
                   object: o,
                 },
               }));
+            })
+            .catch(e => {
+              this.loaded = false;
+              this.dispatchEvent(new MessageEvent('loadFail', {
```

Comment:
Probably a more appropriate event name would be `error`.

Reply:
Updated! :)